Sming/Core/Network/Http/HttpServerConnection.h

```diff
@@ -76,6 +76,11 @@ class HttpServerConnection : public HttpConnection
 		return &request;
 	}
 
+	void setCloseOnBadBodyContent(bool close = true)
```

Comment:
Rename `setCloseOnBadBodyContent` to `setCloseOnContentError`.

Reply:
done